views/js/controller/ItemResultPreviewer/itemResultPreviewer.js

```diff
@@ -24,7 +32,8 @@ define(['taoItems/previewer/factory'], function(previewerFactory) {
             previewerFactory(type, uri, state, {
                 view: 'scorer',
                 fullPage: true,
-                hideActionBars: true
+                hideActionBars: true,
+                plugins
```

Comment:
Does it affect other apps except manual scoring someway? Or is it ok? 🤔

Reply:
no, it doesn't ;) that's the good thing about lti outcome ui, only MS uses it